src/controllers/subject.js

```diff
@@ -0,0 +1,16 @@
+const getMatchOptions = require('~/utils/getMatchOptions')
+const subjectService = require('~/services/subject')
+
+const getNamesByCategoryId = async (req, res) => {
+  const { id: category } = req.params
```

Comment:
I guess you can add check if the parameter is passed, if not, you can send the 400th status code for client